app/src/main/res/drawable/thumb_docker.xml

```diff
@@ -0,0 +1,181 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="270dp"
```

Comment:
Could you resize this to be max of 128x128? Alternatively docker will come in with #398